### ISSUE TITLE
Modify systemd-networkd option during installation

### ISSUE THREAD

**Issue by Torxed** [archlinux/archinstall#1420]
As of soon:tm: the archiso might no longer support `systemd-networkd` as the default network configuration.
This means that our `"Copy ISO network configuration"` won't actually do anything.

Upstream issue tracking this change: https://gitlab.archlinux.org/archlinux/archiso/-/issues/187

**Comment by kaihendry** [comment]
Yes, I was very confused by this option https://youtu.be/Ow09ttLAC2Y?t=802

Though on reboot, systemd-networkd seems to be working as expected: :clap: 

```
$ ls /etc/systemd/network
20-ethernet.network  20-wlan.network  20-wwan.network
```

**Comment by Torxed** [comment]
> Yes, I was very confused by this option https://youtu.be/Ow09ttLAC2Y?t=802

The option itself is assuming you understand how the ISO "got networking" up and running.
Usually this involves either connecting to a wifi and in that case we copy the configuration or you relied on ethernet automatically working and this that option copies the same automatic "magic" :)

I'd prefer if we changed it to "Use best effort systemd-networkd" configuration in the future as the ISO won't do any magic other than running NetworkManager.
